### ISSUE TITLE
remove deprecation warnings

### ISSUE THREAD

**Issue by jasmainak** [glm-tools/pyglmnet#239]
We should remove deprecation warnings like this one:

```py
DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
```

**Comment by pavanramkumar** [comment]
@jasmainak i think this is pretty simple. we just need to run `check_version` in `test_pyglmnet` before importing `cross_validation`, `KFold`, etc. want to take a stab at it?

**Comment by jasmainak** [comment]
yep this issue is mine :) It's simple but requires iteration to get it working I think ...

**Comment by pavanramkumar** [comment]
closed by #263
